app/sass/screen.scss

```diff
@@ -0,0 +1,91 @@
+@import "compass/reset";
+
+body {
+//	padding:15px;
```

Comment:
Soft tabs set to two spaces throughout, remove commented code https://github.com/airbnb/css